### ISSUE TITLE
Release note (candidate) generation of Google Cloud BOM / Libraries BOM

### ISSUE THREAD

**Issue by suztomo** [googleapis/release-please#1290]
@chingor13 

 **Is your feature request related to a problem? Please describe.**

A feature request.

 **Describe the solution you'd like**

I'm thinking to create release note of the Libraries BOM through "DevRel Release Notes Manager". Given a release of Libraries BOM, it would be great if I can get the list of changes in the Google Cloud client libraries that occurred since the last release of the Libraries BOM.

Note that the Libraries BOM imports the Google Cloud BOM. Therefore the notable changes in the client libraries between Google Cloud BOM releases works very well.

 **Describe alternatives you've considered**

I can manually ask the Google library developers for new features they implemented since last release.

 **Additional context**

https://cloud.google.com/ai-hub/docs/release-notes is an example page generated by "DevRel Release Notes Manager".

In general, a good release note has noteworthy user-facing changes and a brief overview of changes, with links to more information.

My current thinking is that the automation would generate candidates for release notes. A human (the releaser of the Libraries BOM) still need to adjust links and words in a customer friendly manner. (If you think automation is not a good idea, let me know your thought.)

**Comment by chingor13** [comment]
The library has no direct link to the "DevRel Release Notes Manager" it's only responsible for helping with version bumping and crafting release notes from commits since the last release.

It seems like the feature request is to expand the release notes of a dependency update commit to include the release notes from that commit.

**Comment by suztomo** [comment]
> expand the release notes of a dependency update commit to include the release notes from that commit.

Yes, the expansion of Google Cloud BOM's release note since the last release is what I'm looking for. This gives an overview when manually writing release note of Libraries BOM.

**Comment by suztomo** [comment]
I started running [a simple Java code](https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/2294/files) to generate Libraries BOM to figure out what I want in the automation in future. Here is the result for its 25.0.0 release:
https://github.com/GoogleCloudPlatform/cloud-opensource-java/releases/tag/v25.0.0-bom

This is just PoC because it's not on release-please bot.

**Comment by bcoe** [comment]
@suztomo 👍 Jeff and I were talking about this functionality this week, let's make sure we can make release-please support your needs.

**Comment by suztomo** [comment]
@bcoe Thank you for the good news. Yesterday during Java sync meeting, there were ideas to add a collapsible sublist for each library to have new features and breaking changes in the same page, and to skip dependency upgrades (they're not interesting to users). I'll refine the ideas further.

Also, as you might hear during Tuesday's meeting, we have a plan to migrate the libraries-bom under googleapis/java-cloud-bom repository.

**Comment by suztomo** [comment]
Closing this in favor of https://github.com/googleapis/release-please/issues/1539

**Comment by suztomo** [comment]
It seems that #1539 is a bit different from this ticket.
